app-admin/pinkybar/metadata.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="project">
+		<email>proxy-maint@gentoo.org</email>
+	</maintainer>
+	<maintainer type="person">
+		<email>sacewe@emailure.net</email>
+		<name>Aaron Caffrey</name>
+	</maintainer>
+	<upstream>
+		<remote-id type="github">su8/pinky-bar</remote-id>
+	</upstream>
+	<use>
+		<flag name="alsa">To get the sound volume level.</flag>
```

Comment:
I presume this works only for ALSA drivers, so please make this explicit in the description. Also, please try to make the descriptions cleaner. Choose one sentence form and use it consistently. Use proper capitalization and writing for abbreviations/acronyms. Use commas and dots appropriately. And most importantly, put yourself in the position of a user who doesn't know the app, wants to try it and needs to select USE flags. The descriptions should clearly let him say 'I want this' or 'I do not want this'.